frontend/src/app/shared/components/button/button.component.sass

```diff
@@ -0,0 +1,43 @@
+@import "/src/styles/color"
+
+.base-button
+    font-family: montserrat-variable-font, sans-serif
+    font-weight: 600 
+    align: center
+    border-radius: 9px
+    cursor: pointer
+    transition: background-color 0.3s, color 0.3s
+
+.action-button
+    color: $link-color
+    background-color: transparent
+    border: 1px solid $link-color
+
+    &:hover
+        background-color: $link-color
+        color: white
+
+.sign-in-button
```

Comment:
I think sign in btn style can be add by default because class name make someone confuse or just change name or you can also create directives for this component